test/tests/functional/pbs_apbasil_inventory.py

```diff
@@ -173,27 +209,37 @@ def test_InventoryQueryVersion(self):
         """
 
         engine_query_log = "<BasilRequest protocol=\"%s\" method=\"QUERY\" \
-type=\"ENGINE\"/>" % (self.basil_version[0])
+type=\"ENGINE\"/>" % (self.available_version)
         self.mom.log_match(engine_query_log, max_attempts=3)
 
-        basil_version_log = "alps_engine_query;The basilversion is set to " + \
-            self.basil_version[0]
+        if self.available_version == '1.7':
+                msg = 'This Cray system supports the BASIL 1.7 protocol'
+                self.mom.log_match(msg, n='ALL', max_attempts=3)
```

Comment:
Since the DEBUG level of this log message has changed in the code, you will have to change the mom log level for the test if the test needs to find this log message.

Reply:
Done